userdetails-gorm/src/main/groovy/au/org/ala/userdetails/gorm/GormUserService.groovy

```diff
@@ -571,7 +573,7 @@ class GormUserService implements IUserService {
     }
 
     @Override
-    User findByUserNameOrEmail(String username) {
+    UserRecord findByUserNameOrEmail(String username) {
```

Comment:
You can leave the return type as User here

Reply:
Fixed